src/platforms/common/enriching-events/identify-user.mdx

```diff
@@ -31,11 +31,18 @@ Your internal identifier for the user.
 
 The username. Typically used as a better label than the internal id.
 
+<PlatformSection supported={["go", "dart"]}>
+
+### `name`
+
+Human readable name of the user.
+</PlatformSection>
```

Comment:
```suggestion
<PlatformSection supported={["go", "dart"]}>

### `name`

The user's first and/or last name.
</PlatformSection>
```

Reply:
Yep, this sounds way better, thank you!
I got this from our realy code, https://github.com/getsentry/relay/blob/e4d11ee25bb1b85eeec9ab4fbdee463248eb5b40/relay-general/src/protocol/user.rs#L59 😬